mintlify/snippets/kyc-onboarding.mdx

```diff
@@ -0,0 +1,632 @@
+- ✅ Provide only essential fields required by UMA providers
+- ✅ Faster customer onboarding process
+
+**No (isRegulatedFinancialInstitution: false)**
+
+- ✅ **Use Hosted KYC Link Flow** for new customers
+- ✅ Let UMA providers handle identity verification
+- ✅ Provide comprehensive customer documentation if using direct API
+</Accordion>
+
+<Accordion title="Additional Considerations">
+**When to use Hosted KYC even as a regulated platform:**
+- **Business customers** (always required)
+- Enhanced verification requirements for specific use cases
+- International customers requiring local compliance
+- High-value transactions requiring additional verification
+- Regulatory requirements for specific customer types
+</Accordion>
+</AccordionGroup>
+
+## Hosted KYC Link Flow
+
+The hosted KYC flow provides a secure, hosted interface where customers can complete their identity verification and onboarding process.
+
+<Note>
+**Primary use case**: Unregulated platforms should use this flow for complete identity verification. Regulated platforms may use this flow for enhanced verification or when required by specific use cases. **Business customers must always use this flow regardless of platform regulatory status.**
+</Note>
+
+### Generate KYC Link
+
+<RequestExample>
+```http GET /customers/kyc-link
+GET /customers/kyc-link?redirect_uri=https://yourapp.com/onboarding-complete
+Authorization: Basic <base64-encoded-credentials>
+```
+</RequestExample>
+
+<ResponseExample>
+```json Success
+{
+  "kycUrl": "https://kyc.lightspark.com/onboard/abc123def456"
+}
+```
+</ResponseExample>
+
+<ParamField query="redirect_uri" type="string">
+Optional URI parameter that specifies where the customer will be automatically redirected after successfully completing the hosted KYC verification process. If not provided, customers will be redirected to a default completion page.
+</ParamField>
+
+<ResponseField name="kycUrl" type="string" required>
+A hosted KYC link for your customer to complete identity verification and onboarding. This URL contains the KYC session and any redirect configuration.
+</ResponseField>
+
+<Note>
+The `redirect_uri` parameter is embedded in the KYC URL and will be used to redirect the customer back to your application after they complete the verification process. This provides a seamless user experience by returning them to your platform.
+</Note>
+
+### Complete KYC Process
+
+<Steps>
+<Step title="Generate KYC Link">
+  Call the `/customers/kyc-link` endpoint with your `redirect_uri` parameter to generate a hosted KYC URL for your customer.
+  
+  <Note>
+  The `redirect_uri` parameter is embedded in the generated KYC URL and will be used to automatically redirect the customer back to your application after they complete verification.
+  </Note>
+</Step>
+
+<Step title="Redirect Customer">
+  Redirect your customer to the returned `kycUrl` where they can complete their identity verification in the hosted interface.
```

Comment:
I'm fine with keeping this as-is for now since we don't know which vendor we're using yet, but I do think there's a good chance we use an iframe based flow like with plaid link.